src/main/java/uk/gov/companieshouse/itemhandler/kafka/OrdersKafkaConsumer.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.companieshouse.itemhandler.kafka;
+
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.kafka.consumer.CHKafkaConsumerGroup;
+import uk.gov.companieshouse.kafka.consumer.ConsumerConfig;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import static java.util.Collections.singletonList;
+import static uk.gov.companieshouse.itemhandler.ItemHandlerApplication.APPLICATION_NAMESPACE;
+
+@Service
+public class OrdersKafkaConsumer implements InitializingBean {
+    private static final Logger LOGGER = LoggerFactory.getLogger(APPLICATION_NAMESPACE);
+    private static final String ORDER_RECEIVED_TOPIC = "order-received";
+    private static final String GROUP_NAME = "order-received-consumers";
+    private CHKafkaConsumerGroup chKafkaConsumerGroup;
```

Comment:
I think we will need to use a `CHKafkaResilientConsumerGroup` to make this a resilient consumer. See for example:

- https://companieshouse.atlassian.net/wiki/spaces/CHS/pages/122890352/Kafka+consumer+resilience
- https://companieshouse.atlassian.net/wiki/spaces/S1/pages/379191300/CH-KAFKA+Java+Consumer+Design
- https://companieshouse.atlassian.net/wiki/spaces/DEV/pages/129564747/Kafka+resilience
- https://companieshouse.atlassian.net/wiki/spaces/PDWC/pages/1254326521/Technical+Requirement+Use+the+CH-Kafka+Library+for+Kafka#TechnicalRequirement:UsetheCH-KafkaLibraryforKafka-Howtousethech-kafkalibraryresiliencepackage

Reply:
I have changed the implementation to add resilience to the consumer.